kolibri/plugins/coach/assets/src/modules/pluginModule.js

```diff
@@ -66,7 +66,7 @@ export default {
       store.commit('SET_DATA_LOADING', true);
       store.commit('SET_CLASS_LIST', []); // Reset the list if we're loading a new one
       return ClassroomResource.fetchCollection({
-        getParams: { parent: facilityId || store.getters.currentFacilityId, role: 'coach' },
+        getParams: { parent: facilityId || store.getters.userFacilityId, role: 'coach' },
```

Comment:
Yes i saw but had no idea how to resolve them . Thanks again!!

Reply:
i do apologise in order to resolve conflicts , i tried to fetch and ultimately it closed this PR .

Reply:
it looks like you may have force-pushed your branch to be completely match `develop`, rather than just the `pluginModule.js` file, which means the changes you intended to introduce here might have gotten lost. you can reopen this PR from its new base, but you may have to reapply the changes you made.

Reply:
yes ,will fix this asap